CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Bug Fixes
 
 - **AWSMobileClient**
+  - Prevent user from cancelling signout after account deletion with hosted UI. (See [PR #3965](https://github.com/aws-amplify/aws-sdk-ios/pull/3965))
```

Comment:
Add a comment saying that this is breaking change, and the reason. You can find example below.

Reply:
done.